misk/src/main/kotlin/misk/web/HttpCall.kt

```diff
@@ -24,15 +24,9 @@ sealed class SocketAddress {
 /**
  * A live HTTP call from a client for use by a chain of network interceptors.
  */
-interface HttpCall {
+interface HttpCall: HttpRequest {
 
-  /** Immutable information about the incoming HTTP request. */
```

Comment:
I think this doc covers the first 3 parameters of the interface and should remain.